src/webview/webview2/mod.rs

```diff
@@ -471,6 +471,13 @@ impl InnerWebView {
       }
     }
 
+    // Set user agent
+    if let Some(user_agent) = attributes.user_agent {
+      unsafe {
+        webview.Settings()?.SetUserAgent(String::from(user_agent.as_str()));
```

Comment:
Looks like I don't have the right to push our PR.
Could you add these lines for me?
```suggestion
        let settings: ICoreWebView2Settings2 = webview.Settings()?
          .cast()
          .map_err(webview2_com::Error::WindowsError)?;
        settings.SetUserAgent(String::from(user_agent.as_str()))?;
```

Reply:
and probably run `cargo fmt` after this

Reply:
Done!